### PR TITLE
Updated AUTHORS to match paper

### LINKED ISSUE #322: Finalise author list here and for the paper
I'm raising this issue to track whether we have finalised the list of authors for the paper, and to make sure that the `AUTHORS` file in this repository is finalised accordingly. This is currently blocking the Zenodo upload (discussed in #318) which will require a final authors list for version 1 before archival.

### PR DESCRIPTION
**Pull request by ml-evs**
It is my understanding that the authors list for this repository should match the authors list in the paper.

This PR brings them in sync for now, with some expected changes over the coming weeks.

This PR should not be used to discuss whether particular authors should be included, instead this should be done by the "group leaders" for each implementation.

Eventually closes #322.

### CODE REVIEW

**Comment by ml-evs** [comment]
I even managed to fail the CI with this PR... at least it spotted a bug in the ordering of our paper's authors!

**Comment by ml-evs** [comment]
As we will soon be submitting the OPTIMADE paper, I have performed the final sync of the AUTHORS file with the author list of the paper, so this should now be ready to merge.

**Comment by rartino** [comment]
I started proofreading this relative to what is in the paper author list right now and I think I am confused.

@ml-evs You say that the point is that the AUTHOR list should match the paper, but already the second name in this list (Archer) is not on the paper? Maybe the paper author list was just recently modified, and this just means we shouldn't merge this PR until the paper is actually submitted?

**Review by gmrigna — APPROVED** [review]

**Comment by gmrigna** [comment]
All the authors listed here are those who attended the meetings. Not all of them are on the paper (e.g. some indicated that they should not be on the paper). In contrast, some of the authors of the paper were not here. I guess that the fix is about the latter ones. I would think that all the participants and the authors should be listed here. Being the author of a paper is a different topic, don't you think @rartino

**Comment by rartino** [comment]
@gmrigna Hmm... I made my comment just on the basis of recalling us agreeing on what @ml-evs says in the top post: *"the authors list for this repository should match the authors list in the paper."* That approach made sense to me when the author list was explicitly divided into 'paper authors' and 'optimade v1.0 contributors'. However, perhaps things are less clear now with the paper author list divided into "equal contribution authors" and "other authors". 

On the other hand, the list of names in this file is placed under the headline "The OPTIMADE development team" which I read quite much as resonating with a sensible criteria for inclusion on the paper. It seems odd to claim someone to be part of the "development team" while not having contributed enough to be on the author list.

So, here is a suggestion. How about:
1. We place the headline "The OPTIMADE development team in alphabetical order" at the very top of this file, and include exactly all names of the paper under that headline (but in alphabetical order). The criteria to be added to this list in the future is having supplied a concrete contribution to OPTIMADE. As a general rule, publications on future versions of OPTIMADE should offer these names co-authorship. 
2. We let that section *be followed by*:
```
The OPTIMADE API has been developed with contributions by the participants of the workshops "Open Database Integration for Materials Design", held at:
- the Lorentz Center in Leiden, Netherlands from 2016-10-24 to 2016-10-28
- the CECAM in Lausanne, Switzerland from 2018-06-11 to 2018-06-15
- the CECAM in Lausanne, Switzerland from 2019-06-11 to 2019-06-14
- the CECAM in Lausanne, Switzerland (virtually) 2020-06-08 to 2020-06-12

The participants of these workshops in alphabetical order:
<the full list we presently have>
```
(perhaps also updating that list to include all the names of people present in the last workshop.)

**Comment by ml-evs** [comment]
> @gmrigna Hmm... I made my comment just on the basis of recalling us agreeing on what @ml-evs says in the top post: _"the authors list for this repository should match the authors list in the paper."_ That approach made sense to me when the author list was explicitly divided into 'paper authors' and 'optimade v1.0 contributors'. However, perhaps things are less clear now with the paper author list divided into "equal contribution authors" and "other authors".

Yes, when I first made this PR and wrote the top comment (September 2020), no-one had recused themselves from the paper... you are right that this is now paper+workshops. I toyed with changing the wording myself, but decided it wasn't worth it...


> So, here is a suggestion. How about:

I don't mind either way; I'm not against a the flat contributors list as it is not meant to indicate the size of the contribution, but if people prefer to change it then they should have access to modify this PR.

**Comment by gmrigna** [comment]
Maybe what @rartino  suggests sticks to facts: 1) who contributed and 2) who participated to the workshops. I am fine to go along that line...

File: AUTHORS [hunk 1]
```diff
@@ -4,6 +4,7 @@ Design", held at:
 - the Lorentz Center in Leiden, Netherlands from 2016-10-24 to 2016-10-28
 - the CECAM in Lausanne, Switzerland from 2018-06-11 to 2018-06-15
 - the CECAM in Lausanne, Switzerland from 2019-06-11 to 2019-06-14
+- the CECAM in Lausanne, Switzerland (virtually) 2020-06-08 to 2020-06-12
 
 The OPTIMADE development team in alphabetical order:
```
Comment (rartino) [inline AUTHORS:10]:
```suggestion
The OPTIMADE API has been developed with contributions by the participants of the workshops "Open Database Integration for Materials Design", held at:
- the Lorentz Center in Leiden, Netherlands from 2016-10-24 to 2016-10-28
- the CECAM in Lausanne, Switzerland from 2018-06-11 to 2018-06-15
- the CECAM in Lausanne, Switzerland from 2019-06-11 to 2019-06-14
- the CECAM in Lausanne, Switzerland (virtually) 2020-06-08 to 2020-06-12

The participants of these workshops in alphabetical order:
```

**Review by rartino — COMMENTED** [review]
Here is a concrete suggestion; I propose that for now we just slightly update the formulation at the top of the file to get away from the "development team" phrasing which (at least IMO) clashes a bit with the existence of a paper with only a subset of these as authors. However, since the names are kept the same (except for the fixes provided by @ml-evs) this isn't a major change.

Then, next OPTIMADE web meeting we bring this up and check how the others feel about adding an additional header to this file that separates a "development team" more in line with the paper author list, like I suggested above.

File: AUTHORS (hunk 1 above)
Comment (gmrigna) [reply AUTHORS:10]: Agreed!

Comment (ml-evs) [reply AUTHORS]: I completely missed this suggestion, apologies @rartino

File: AUTHORS [hunk 2]
```diff
@@ -1,60 +1,69 @@
...
 Design", held at:
+The OPTIMADE API has been developed with contributions by the participants of the workshops "Open Database Integration for Materials Design", held at:
 - the Lorentz Center in Leiden, Netherlands from 2016-10-24 to 2016-10-28
 - the CECAM in Lausanne, Switzerland from 2018-06-11 to 2018-06-15
 - the CECAM in Lausanne, Switzerland from 2019-06-11 to 2019-06-14
+- the CECAM in Lausanne, Switzerland (virtually) 2020-06-08 to 2020-06-12
 
-The OPTIMADE development team in alphabetical order:
-
+The participants of these workshops in alphabetical order:
```
Comment (ml-evs) [inline AUTHORS:10]:
```suggestion
Contributors to the development of the OPTIMADE specification and implementation in alphabetical order:
```

**Review by gmrigna — APPROVED** [review]

**Comment by ml-evs** [comment]
@rartino We have just done one final tweak to the wording, are you happy with this now?

File: AUTHORS [hunk 3]
```diff
@@ -1,60 +1,69 @@
 The initial release of the OPTIMADE API was developed by the
 participants of the workshops "Open Databases Integration for Materials
 Design", held at:
+The OPTIMADE API has been developed with contributions by the participants of the workshops "Open Database Integration for Materials Design", held at:
```
Comment (rartino) [inline AUTHORS:4]: I don't understand, now the top "header" looks duplicated? Isn't this what you intend?:
```suggestion
The OPTIMADE API has been developed with contributions by the participants of the workshops "Open Database Integration for Materials Design", held at:
```

**Review by rartino — CHANGES_REQUESTED** [review]

File: AUTHORS (hunk 3 above)
Comment (ml-evs) [reply AUTHORS:4]: Ah, it looks like the suggestion of yours I committed was missing the first line - my fault!

Comment (ml-evs) [reply AUTHORS]: and now we are back in the loop of requiring two reviewers :weary:

**Review by rartino — APPROVED** [review]

**Review by gmrigna — APPROVED** [review]

**Pull request merged**